src/cookbook/design/orientation.md

```diff
@@ -135,7 +135,41 @@ class OrientationList extends StatelessWidget {
   <img src="/assets/images/docs/cookbook/orientation.gif" alt="Orientation Demo" class="site-mobile-screenshot" />
 </noscript>
 
+## Locking device orientation
+
+In the previous section you have learned 
+how to adapt the app UI to device orientation changes.
+
+However, Flutter also allows you define the specific preferred orientations
+that your app will support.
+This allows you to lock the app to single orientation 
+(e.g. only portrait up position)
+or to allow multiple orientations 
+(e.g. both portrait up and down, but not lanscape).
```

Comment:
**issue (blocking):** Tighten the language, fix verb tense, remove parentheticals, and fix spelling mistake.

```suggestion
Flutter also allows you define the specific preferred orientations
that your app supports. You can either:

- lock the app to single orientation like only portrait up position or
- allow multiple orientations like both portrait up and down, but not landscape.
```